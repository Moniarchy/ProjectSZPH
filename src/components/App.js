import React, { Component } from 'react';
import ShipConsole from './ShipConsole'
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="app">
        <ShipConsole />
      </div>
    );
  }
}

export default App;
