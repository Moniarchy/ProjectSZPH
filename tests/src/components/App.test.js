import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import App from '../../../src/components/App'

describe( 'App component', () => {
  const appWrapper = shallow( <App /> )

  it( 'can be snapshot', () => {
    const appSnapshot = renderer.create( <App /> ).toJSON()
    expect( appSnapshot ).toMatchSnapshot()
  })

  it( 'renders without crashing', () => {
    const appDiv = appWrapper.find( '.app' )
    expect( appDiv.type() ).toEqual( 'div' )
  })

})
