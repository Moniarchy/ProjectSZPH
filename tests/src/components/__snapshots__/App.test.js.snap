exports[`App component can be snapshot 1`] = `
<div
  className="app">
  <div
    className="app-header">
    <img
      alt="logo"
      className="app-logo"
      src="test-file-stub" />
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    className="app-introduction">
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
